### PR TITLE
NEWS for v244

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

**Comment by yuwata** [comment]
LGTM. Let's merge this.

File: NEWS [hunk 1]
```diff
@@ -1,5 +1,154 @@
...
+
+        * udev now provides a program (fido_id) that identifies FIDO CTAP1
+          ("U2F")/CTAP2 security tokens based on the usage declared in their
+          report and descriptor and outputs suitable environment variables.
+          This replaces the externally maintained whitelists of all known
+          security tokens that were used previously.
+
+        * Automatically generated autosuspend udev rules for whitelisted
+          devices have been imported from the Chromium OS project. This should
+          improve power saving with of many more devices.
```
Comment (mbiebl) [inline NEWS:55]: typo "with of"

File: NEWS [hunk 2]
```diff
@@ -1,5 +1,154 @@
 systemd System and Service Manager
 
+CHANGES WITH 243 in spe:
```
Comment (michaelolbrich) [inline NEWS:3]: 244 right?

File: NEWS [hunk 3]
```diff
@@ -1,5 +1,154 @@
...
+          the device in exclusive mode as they should.
+
+        * systemd-networkd does not create a default route for IPv4 link local
+          addressing anymore. The creation of the route was unexpected and was
+          breaking routing in various cases, but people who rely on it being
+          created implicitly will need to adjust. Such a route may be requested
+          with DefaultRouteOnDevice=yes.
+
+          Similarly, systemd-networkd will not assign a link-local IPv6 address
+          when IPv6 link-local routing is not enabled.
```
Comment (yuwata) [inline NEWS:74]: Maybe, `when IPv6 link-local addressing is not enabled`?

Comment (keszybz) [reply NEWS:74]: I used "routing" because "address" appears already in the sentence...

File: NEWS (hunk 2 above)
Comment (keszybz) [reply NEWS:3]: Fixed in master now.

File: NEWS (hunk 1 above)
Comment (keszybz) [reply NEWS:55]: Fixed in master now.

**Pull request merged**